kubectl/internal/commands/applybuild.go

```diff
@@ -0,0 +1,97 @@
+package commands
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	"k8s.io/klog/v2"
+
+	"github.com/spf13/cobra"
+	"github.com/substratusai/substratus/kubectl/internal/client"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+func ApplyBuild() *cobra.Command {
+	var cfg struct {
+		filename   string
+		build      string
+		kubeconfig string
+	}
+
+	var cmd = &cobra.Command{
+		Use:   "applybuild [flags]",
+		Short: "Apply a Substratus object, upload and build container in-cluster from a local directory",
```

Comment:
@brandonjbjelland What do you think about replacing the existing `kubectl build remote` command with `kubectl applybuild`?

I see this as an extension of `kubectl apply` (for Substratus objects) where you don't need to first build an push a docker container.

Reply:
`applybuild` makes good sense to me - it captures the action a bit better without being too lengthy (`applyremotebuild` is probably most accurate but 👎 )